lib/iso_builder.py

```diff
@@ -89,6 +89,7 @@ def __init__(self, config):
         (_, _, self.arch) = distro_utils.detect_distribution()
         self.pungi_binary = self.config.get('pungi_binary') or "pungi"
         self.pungi_args = self.config.get('pungi_args') or ""
+        self.build_iso = self.config.get('iso') or False
```

Comment:
The `or False` is kind of redundant. It is needed in the other cases because if the config doesn't exist, `None` will be returned, and passing `None` when a string is expected leads to strange behavior (`None` is converted to `"None"`).
Passing `None` instead of a boolean is common in python, though.
If you want to leave it to be more explicit, no problem. Another option would be `bool(self.config.get('iso'))`

Reply:
You could mention in the commit message that this commit is only preparing for the addition of other build-images options. As I understand it, this commit is not altering any behavior.